src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const key in robot) {
+    const prop = robot[key];
+
+    if (inverse[prop]) {
```

Comment:
Please check paragraph 2 in checklist
This property presence check is not so good